QP_solver/test/QP_solver/master_mps_to_derivatives.cpp

```diff
@@ -58,6 +68,20 @@ namespace QP_from_mps_detail {
   struct MPS_type_name<int> {
     static const char *name() { return "integer"; }
   };
+#ifdef CGAL_USE_BOOST_MP
+  template <class Backend, boost::multiprecision::expression_template_option Eto>
+  struct MPS_type_name<boost::multiprecision::number<Backend, Eto> > {
+    typedef boost::multiprecision::number<Backend, Eto> NT;
+    static const char *name() {
+      if (boost::multiprecision::number_category<NT>::value == boost::multiprecision::number_kind_integer)
+	return "integer";
+      else if (boost::multiprecision::number_category<NT>::value == boost::multiprecision::number_kind_rational)
+	return "rational";
+      else
+	return 0;
```

Comment:
This file is already full of TABs. Let's ignore this issue with the Github display, for this file.

But not for `Number_types/include/CGAL/boost_mp.h`, because it is a new file.